examples/oas3/petstore-with-oidc-kuadrant-extensions.yaml

```diff
@@ -0,0 +1,55 @@
+---
+openapi: "3.0.3"
+info:
+  title: "Pet Store API"
+  version: "1.0.0"
+  x-kuadrant:
+    route:
+      name: "petstore"
+      namespace: "petstore"
+      hostnames:
+        - example.com
```

Comment:
Not getting from `servers`?

Reply:
yes, it could be. But I can see it broken because some host rewrite happens for some reasons. It should default to `servers` and be overridden somehow IMO. I will implement reading from servers and override from `hostnames` if the field exists.